celeri/get_mesh_edge_elements.py

```diff
@@ -0,0 +1,56 @@
+def get_mesh_edge_elements(meshes):
+    # Find indices of elements lining top, bottom, and sides of each mesh
+    
+	from ismember import ismember
+	
+	celeri.get_ordered_edge_nodes(meshes)
+	
+	for i in range(meshes):
```

Comment:
Throughout we're going with an indentation of 4 spaces at each level.  It looks like more here?

Reply:
Edited (saved with black formatting) in a876f1233bb3b0e1b63337fd2e8b9800816d7e6c